### DISCUSSION TITLE
talk about evomaster.jar

### DISCUSSION THREAD

**Discussion by errors11** [EMResearch/EvoMaster#1015] [Q&A]
The open source version of EvoMaster is a combination of white box and black box, and can you publish a white-box code, that is, the source code of the evomaster.jar?

**Comment by arcuri82** [comment]
hi,
 I am unsure I understand your question.

All code, for both black and white box testing, is under https://github.com/WebFuzzing/EvoMaster

On [release page](https://github.com/WebFuzzing/EvoMaster/releases), there is also the `Source code` in zip and tar.gz format

**Reply by errors11** [reply]
Sorry,I mean I need the white box part code, thanks for the reply.

**Comment by errors11** [comment]
And，If I want to improve this tool towards web security goals, such as using it to detect SQL injections, etc., do you have any good suggestions?
